databricks/koalas/tests/test_indexes.py

```diff
@@ -777,39 +777,40 @@ def test_monotonic(self):
         datas.append([(-5, 'e'), (-4, 'c'), (-3, 'b'), (-2, 'd'), (-1, 'a')])
 
         # None type tests (None type is treated as the largets value)
-        datas.append([(None, 100), (2, 200), (3, 300), (4, 400), (5, 500)])
+        # TODO: the commented tests below should be uncommented after fixing for pandas >= 1.0.0
```

Comment:
Is it difficult to patch to pandas' 1.0.0 behaviours?

Reply:
yeah its changing looks a bit complicated, let me try to fix this follow-up PR